Yosemite/Yosemite/Stores/ReceiptStore.swift

```diff
@@ -56,4 +62,66 @@ private extension ReceiptStore {
         let renderer = ReceiptRenderer(content: content)
         onContent(renderer.htmlContent())
     }
+
+    func loadReceipt(order: Order, onCompletion: @escaping (Result<CardPresentReceiptParameters, Error>) -> Void) {
+
+        guard let outputURL = try? fileURL(order: order) else {
+            let error = ReceiptStoreError.fileNotFound
```

Comment:
When I was testing this earlier I went into the wrong order and opened one that didn't have a receipt and the error wasn't coming from here, but from the decoder not being able to load data at the given URL. I think the `fileURL` method will succeed as long as the URL is valid regardless of the file existing or not. Maybe we want to add an extra check for that, if we care about differentiating missing receipts from wrong ones.

Reply:
Good catch, thanks!